ml_genn/ml_genn/compilers/event_prop_compiler.py

```diff
@@ -554,6 +554,10 @@ def pre_compile(self, network: Network,
         readouts = [p for p in network.populations
                     if p.neuron.readout is not None]
 
+        if not readouts:
+            raise ValueError("EventPropCompiler needs an output - "
+                             "No readout populations found in the network.")
```

Comment:
I think we'd normally use a runtime error here
```suggestion
            raise RuntimeError("EventPropCompiler needs an output - "
                               "No readout populations found in the network.")
```

Reply:
ok, I gonna fix it